src/argilla/client/workspaces.py

```diff
@@ -177,6 +177,14 @@ def delete_user(self, user_id: str) -> None:
                 f"Error while deleting user with id=`{user_id}` from workspace with id=`{self.id}`."
             ) from e
 
+    def delete(self):
```

Comment:
Here you will probably need to add `@allowed_for_roles(roles=[UserRole.owner])` once you merge `develop` into your branch, as this is part of #3383